src/lib/utils/relay/index.ts

```diff
@@ -16,5 +16,10 @@ export function formatRelayName(relay: NDKRelay): string {
     // strip protocol prefix
     name = name.replace(/^(ws|wss):\/\//, '');
 
+    // truncate long urls
+    // e.g. filter.nostr.wine/{npub}?broadcast=true
+    if (name.length > 28) {
```

Comment:
probably better to do it with CSS?

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
I was trying to leave the styling as minimal as possible. Maybe we just shouldn't truncate at all and let the use pass in styling themselves?

Reply:
Ah, an issue with that is that the user won't have an easy way to target the HTML element that houses the relay URL specifically.